src/main/java/uk/gov/register/configuration/RegisterContentPages.java

```diff
@@ -1,14 +1,17 @@
 package uk.gov.register.configuration;
 
+import java.util.List;
 import java.util.Optional;
 
 public class RegisterContentPages {
     private final Optional<String> registerHistoryUrl;
     private final Optional<String> custodianName;
+    private final Optional<List<String>> similarRegisters;
 
-    public RegisterContentPages(Optional<String> registerHistoryUrl, Optional<String> custodianName) {
+    public RegisterContentPages(Optional<String> registerHistoryUrl, Optional<String> custodianName, Optional<List<String>> similarRegisters) {
```

Comment:
Should we rename this or should we create a separate class for the things which aren't links to content pages?